### ISSUE TITLE
returning nil as UGen graph returns uninformative error

### ISSUE THREAD

**Issue by telephon** [supercollider/supercollider#1771]
```
{ var a, b; a =  SinOsc.ar(100); [a, b] }.plot
```

Returns:

```
ERROR: Message '<' not understood.
RECEIVER:
   nil
ARGS:
   Symbol 'audio'
PATH: /Volumes/data/sc/(this)/__projekte/Pseudomorph/pseudomorph-01.scd

PROTECTED CALL STACK:
	Meta_MethodError:new	0x10f2470c0
		arg this = DoesNotUnderstandError
		arg what = nil
		arg receiver = nil
	Meta_DoesNotUnderstandError:new	0x10f249080
		arg this = DoesNotUnderstandError
		arg receiver = nil
		arg selector = <
		arg args = [ audio ]
	Object:doesNotUnderstand	0x10ea5f280
		arg this = nil
		arg selector = <
		arg args = nil
	a FunctionDef	0x1100d03c0
		sourceCode = "<an open Function>"
		arg elem = nil
		arg i = 1
	ArrayedCollection:do	0x1101a6240
		arg this = [ audio, nil ]
		arg function = a Function
		var i = 1
	Collection:minItem	0x1100cfac0
		arg this = [ audio, nil ]
		arg function = nil
		var minValue = nil
		var minElement = audio
	a FunctionDef	0x110b8a840
		sourceCode = "<an open Function>"
		arg bufnum = an OutputProxy
		var val = [ a SinOsc, nil ]
	SynthDef:buildUgenGraph	0x110938400
		arg this = a SynthDef
		arg func = a Function
		arg rates = nil
		arg prependArgs = [  ]
		var result = nil
		var saveControlNames = nil
	a FunctionDef	0x110936a40
		sourceCode = "<an open Function>"
	Function:prTry	0x10f4c9a40
		arg this = a Function
		var result = nil
		var thread = a Routine
		var next = nil
		var wasInProtectedFunc = true
	
CALL STACK:
	DoesNotUnderstandError:reportError   0x111588f68
		arg this = <instance of DoesNotUnderstandError>
	< closed FunctionDef >   0x115ffed28
		arg error = <instance of DoesNotUnderstandError>
	Integer:forBy   0x111513098
		arg this = 0
		arg endval = 0
		arg stepval = 2
		arg function = <instance of Function>
		var i = 0
		var j = 0
	SequenceableCollection:pairsDo   0x10ec82558
		arg this = [*2]
		arg function = <instance of Function>
	Scheduler:seconds_   0x1120ff388
		arg this = <instance of Scheduler>
		arg newSeconds = 28024.220731268
	Meta_AppClock:tick   0x111dfad88
		arg this = <instance of Meta_AppClock>
		var saveClock = <instance of Meta_SystemClock>
	Process:tick   0x111d68bd8
		arg this = <instance of Main>
^^ The preceding error dump is for ERROR: Message '<' not understood.
RECEIVER: nil
```

**Comment by timblechmann** [comment]
agreed, the error message should be improved, but it should still yield an error.

**Comment by telephon** [comment]
The proposed fix in #1835 lets the Out UGen do the warning, which is the most generally seen and understood error.
